src/components/ItemsList.js

```diff
@@ -1,38 +1,72 @@
 import React from 'react';
 import firebase from '../lib/firebase';
 import Nav from './Nav';
+import '../styles/ItemsList.css';
 import { useCollectionData } from 'react-firebase-hooks/firestore';
 
 const db = firebase.firestore().collection('shopping_list');
 
+const wasItemPurchasedWithinLastOneDay = (lastPurchasedOn) => {
+  if (lastPurchasedOn === null) return false;
+  const oneDayInMilliseconds = 24 * 60 * 60 * 1000;
```

Comment:
Great job on making this a constant instead of just using a [magic number]!(https://en.wikipedia.org/wiki/Magic_number_(programming)#:~:text=The%20term%20magic%20number%20or,1%20manuals%20of%20the%201960s.)